src/commands/fun/carol.js

```diff
@@ -0,0 +1,14 @@
+const { client } = require('../../core/twitch_client');
+
+const REPEAT_AMOUNT = 30;
+const REPEAT_CONTENT = 'carol';
+
+module.exports = {
+  keyword: 'carol',
+
+  async execute({ channel }) {
+    const answer = Array(REPEAT_AMOUNT).fill(REPEAT_CONTENT).join(' ');
```

Comment:
@levxyca prefiro `'carol '.repeat(30)`, mas você que manda.
Assim também funciona.

Reply:
@rn4n `'carol '.repeat(30)` vai adicionar um espaço no final da string

Reply:
É indiferente pro comportamento do bot, tem outros comandos que deixam um espaço no final.
Mas se o problema for um espaço em branco no final, basta usar um [`String#trimEnd`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trimEnd):

```js
 'carol '.repeat(30).trimEnd()
```

Pra mim tanto faz, ambas as soluções fazem o que precisa ser feito.
Só prefiro trabalhar com string diretamente ao invés de criar uma array e passar para string depois. @levxyca por mim ta aprovado.